collectors/node.d.plugin/snmp/README.md

```diff
@@ -183,37 +183,50 @@ The `options` given for each server, are:
 
 To use SNMPv3:
 
--   set `version` to 3
 -   use `user` instead of `community`
+-   set `version` to 3
 
 User syntax:
 
 ```json
 {
-    "user": {
+  "enable_autodetect": false,
+  "update_every": 10,
+  "servers": [
+    {
+      "hostname": "10.11.12.8",
+      "user": {
         "name": "userName",
         "level": 3,
         "authProtocol": "3",
         "authKey": "authKey",
         "privProtocol": "2",
         "privKey": "privKey"
+      },
+      "update_every": 10,
+      "options": {
+        "version": 3
+      },
+      "charts": {
+      }
     }
+  ]
 }
```

Comment:
```JSON
{
    "enable_autodetect": false,
    "update_every": 10,
    "servers": [
        {
            "hostname": "10.11.12.8",
            "user": {
                "name": "userName",
                "level": 3,
                "authProtocol": "3",
                "authKey": "authKey",
                "privProtocol": "2",
                "privKey": "privKey"
            },
            "update_every": 10,
            "options": {
                "version": 3
            },
            "charts": {}
        }
    ]
}
```

Reformatted to match other examples, they use 4 space tab.

Reply:
Thank you! Nice attention to detail. 👍